src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -33,6 +33,7 @@ object JesBackend {
   lazy val GoogleApplicationName = JesConf.getString("applicationName")
   lazy val EndpointUrl = new URL(JesConf.getString("endpointUrl"))
   lazy val JesConnection = JesInterface(GoogleApplicationName, EndpointUrl)
+  lazy val DockerHubCredentials = Try(JesConf.getString("dockerhubCredentialsPath")).toOption
```

Comment:
This was my quick-n'-dirty suggestion last night, but something that doesn't sweep real exceptions into a `None` result might be preferable:

```scala
  implicit class EnhancedConfig(val config: Config) extends AnyVal {
    def getStringOption(key: String): Option[String] = {
      Try(config.getString(key)) match {
        case Success(value) => Option(value)
        case Failure(e: ConfigException.Missing) => None
        case Failure(e) => throw e
      }
    }
  }
  lazy val DockerHubCredentials = JesConf.getStringOption("dockerhubCredentialsPath")
```